src/blocks/map/style.scss

```diff
@@ -106,3 +134,131 @@
 		margin-bottom: 0;
 	}
 }
+
+// These general non-layout styles are sandboxed to the side-by-side mode
+// as of July 2025. They could be expanded to apply to all maps if it is
+// acceptable to retroactively change design on e.g. old annual reports.
+.wp-block-wmf-reports-map.is-style-side-by-side {
+	.marker.active {
+		outline: 4px solid white;
+		outline-offset: -6px;
+	}
+
+	// These controls aren't accessible on the <p> block yet.
+	.wmf-pattern-reports-carousel-slide__location {
+		font-weight: bold;
+		font-style: italic;
+		margin-bottom: 1rem;
+	}
+
+	.wp-block-buttons,
+	.is-style-sans-p {
+		font-size: clamp(1rem, 1vw + 0.5rem, 1.25rem);
+	}
+}
+
+.wp-block-wmf-reports-map.is-style-side-by-side {
+
+	@media screen and (max-width: 719px /* 45rem - 1px */) {
+		.is-layout-constrained > &:not(.alignfull) {
+			// Always full-bleed on small viewports. Override core style.
+			margin-left: calc(-1 * var(--wp--style--root--padding-left)) !important;
+			margin-right: calc(-1 * var(--wp--style--root--padding-right)) !important;
+		}
+	}
+
+	@media screen and (min-width: 45rem) {
+		grid-template-columns: 70% 30%;
+		grid-template-rows: 1fr;
+		grid-template-areas: "main sidebar";
+		align-items: stretch;
+		width: 100%;
+		position: relative;
+		margin: auto;
+
+		#map {
+			background-color: #f2f2f2;
+			min-height: 100%;
+			grid-area: main;
+		}
+
+		.carousel__carousel-wrapper {
+			grid-area: sidebar;
+
+			> * {
+				margin: 1.5rem;
+			}
+		}
+
+		.wmf-pattern-reports-carousel-slide {
+			> .wp-block-columns {
+				display: flex;
+				flex-direction: column;
+				gap: 1.5rem;
+				justify-content: flex-start;
+			}
+
+			&,
+			&.wp-block-group.has-background {
+				padding: 1.5rem;
+			}
+
+			> .wp-block-columns > .wp-block-column > *:not(:last-child) {
+				margin-top: 0;
+				margin-bottom: 1rem;
+			}
+
+			.wp-block-image img {
+				border-radius: 0.25rem;
+			}
+
+			> .wp-block-columns {
+				flex-direction: column;
+			}
+		}
+	}
+
+	.carousel__carousel-wrapper {
+		// Stack all cards on top of each other.
+		display: grid;
+		grid-template-rows: 1fr;
+		grid-template-columns: 1fr;
+		padding-bottom: 3rem;
+	}
+
+	.wp-block-wmf-reports-marker {
+		margin: 0;
+
+		> .alignfull {
+			margin: 0;
+		}
+
+		> :not(.alignfull) {
+			margin: var(--wp--style--root--padding-left, 1rem);
+
+			@media screen and (min-width: 45rem) {
+				margin: 1.5rem;
+			}
+		}
+	}
+
+	&.carousel--uninitialized .wp-block-wmf-reports-marker:not(:first-of-type) {
+		// Avoid CLS when loading in. May be better to switch to a .nojs class?
+		display: none;
+	}
+
+	.carousel-buttons {
```

Comment:
I didn't dig into why (I can if you need me to), but the button position is weird on wider screens.

https://github.com/user-attachments/assets/9e8c787f-7ac3-4d67-a51c-75f4e28aa5db